packages/models/src/eserviceEvents.ts

```diff
@@ -0,0 +1,96 @@
+import z from "zod";
+import { EService, Descriptor, Document } from "./eservice.js";
+
+export const EServiceAdded = z.object({
+  eService: EService,
```

Comment:
Let's keep the rename you did for this PR and think later (when adding protobuf) whether to revert.

It could work to use the new names in the protobuf too, since they won't affect binary serialization.

However the notification service at least relies on events serialized to JSON and would be affected by the rename.